libs/ui/src/bmd_paper_ballot.tsx

```diff
@@ -39,12 +39,50 @@ import {
   NumberString,
 } from './ui_strings';
 
-const Ballot = styled.div`
+export type BmdBallotPrintType = 'vxMark' | 'vsap';
```

Comment:
It might be clearer to have these be `mark` vs `mark-scan` to correspond with the names of the apps in the rest of the codebase

Reply:
Ah, good call